src/Accelerators/NNPA/Conversion/ONNXToZHigh/PerfModelArch14.inc

```diff
@@ -0,0 +1,337 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------- Auto-Generated, do not change  ---------------------===//
+//
+// Copyright 2023-2025 The IBM Research Authors.
+//
+// =============================================================================
+
+// Operation Add_3ds: estimated times.
+static static double arch14_estimatedTimeForCPU_Add_3ds(
```

Comment:
nit: double static

Reply:
fixed.

Reply:
Thanks! Sorry for the confusion, I meant duplicated keyword `static`, not the order of `static` and `double`.